packages/pwa-kit-dev/jest.config.js

```diff
@@ -12,9 +12,9 @@ module.exports = {
     coverageThreshold: {
         global: {
             branches: 67,
-            functions: 67,
-            lines: 70,
-            statements: 70
+            functions: 75,
+            lines: 75,
+            statements: 75
```

Comment:
I understand we'd like to have more coverage but could this threshold change be in a separate PR?

Reply:
If we don't uplift as we go, we're probably never going to get around to bumping the thresholds...

Reply:
I'm good to bump this in a PR. The key question in "should this be a different PR?" is "Will we create a new GUS ticket for this?" 

There will (in my opinion) always be a level of task granularity that falls to the side as "too small for a GUS ticket", I think it's acceptable here